TradeItIosTicketSDK2/Constants.swift

```diff
@@ -67,6 +67,7 @@ public struct TradeItNotification {
         case trading
         case preview
         case submitted
+        case selectActionType
```

Comment:
Can you add this to the objective-c mapping above in `TradeItNotificationConstants`

Reply:
@jsom done